### PR TITLE
various improvements to helm chart

### PR DESCRIPTION
**Pull request by draeron**
- added configuration of LDAP
- added log level configuration
- added service annotations
- added storage class specification
- fix bad registry configmap : https://github.com/kubic-project/caasp-services/issues/37

### CODE REVIEW

**Comment by stefsuse** [comment]
This is awesome, thank you for looking into this, especially ConfigMap! 

Since this is a larger change and there were some PRs before you I would like to get some of those merged before I do a full review. 

One note for now - persistent storage is disabled by default in community charts unless it is absolutely necessary for the app to function.

**Comment by draeron** [comment]
i've reviewed and reverted some default settings i changed by error (the persistence setting was one of them).

File: contrib/helm-charts/portus/values.yaml
Comment (stefsuse) [inline contrib/helm-charts/portus/values.yaml]: uncomment?

File: contrib/helm-charts/portus/values.yaml
Comment (stefsuse) [inline contrib/helm-charts/portus/values.yaml]: remove newline

**Review by stefsuse — CHANGES_REQUESTED** [review]
Overall looks great. I would like to see NOTES.txt updated to include LDAP instructions when applicable, but I won't hold up changes over it.

Please rebase and increment version.

**Comment by draeron** [comment]
i've made both correction, rebased and squashed the commits

i wouldn't mind adding a section in `notes.txt` but honestly i'm not sure what should be said about ldap in there as the usage ain't much different. I guess replacing the _Create admin user account_ with alternate text saying the first login will be promoted to admin account ?

**Comment by stefsuse** [comment]
notes.txt should include basic information of what was deployed and how to get started (it does now) and important information that the chart can't do for you automatically. Some charts will also give a report of any add-ons you enabled. 

Like I said it was just a suggestion if anything came to mind that could help someone get started using the feature. I guess it is normally implied that the first user created is an admin.

**Review by stefsuse — APPROVED** [review]
LGTM and tests passed (sorry for delay on CI)

**Pull request merged**
